experimental/dds/tree2/src/test/shared-tree/undo.spec.ts

```diff
@@ -39,21 +39,24 @@ describe("Undo", () => {
 		expectJsonTree(tree, []);
 	});
 
-	it("the insert of a node on a fork", async () => {
+	it("the insert of a node on a fork", () => {
 		const tree = makeTree("A");
 		const fork = tree.fork();
+		const fork2 = fork.fork();
```

Comment:
There is a helper called `itView` at the bottom of sharedTree.spec.ts which makes it easy to write a test like this that goes through both the "root" `SharedTreeView` and also a first `SharedTreeFork`. You could pull that out into a utils file in the shared-tree folder and use it here too. If you did that, it would clean up these tests and clearly communicate what the goal of doing these forks is. Also, if we ever figure out how to merge those implementations (by making the root local branch a true branch) and there are no longer divergent code paths, it will be really easy to find all the places that are doing this "double fork" pattern and undo it.